### PR TITLE
Docs improvements and logging debug issue #1

### PR DESCRIPTION
**Pull request by Fabio-RibeiroB**
Changes:
1. Changed readme files
2. Added logging.debug lines to try and identify the issue with readwrite function in the /mail route.

### CODE REVIEW

File: docker-compose.yml
Comment (Fabio-RibeiroB) [inline docker-compose.yml:1]: Not sure what happened here with the $

Comment (Fabio-RibeiroB) [reply docker-compose.yml:1]: Maybe there was a version issue

File: dist/myenv-dist.env
Comment (Fabio-RibeiroB) [inline dist/myenv-dist.env:1]: Maybe should change DEFAULT_CONTACT to "YOUR.EMAIL@ADDRESS.COM" or something so you don't get emailed by accident.

Comment (bradymd) [reply dist/myenv-dist.env:1]: Oh yes good point .. wil do

**Comment by bradymd** [comment]
Thanks for getting innvolved

**Pull request merged**